core/processor/src/main/java/io/quarkus/annotation/processor/documentation/config/model/Extension.java

```diff
@@ -59,8 +59,7 @@ public boolean equals(Object obj) {
     @Deprecated(forRemoval = true)
     @JsonIgnore
     public boolean isMixedModule() {
-        return "io.quarkus".equals(groupId) && ("quarkus-core".equals(artifactId) || "quarkus-vertx-http".equals(artifactId)
-                || "quarkus-messaging".equals(artifactId));
+        return "io.quarkus".equals(groupId) && ("quarkus-core".equals(artifactId) || "quarkus-vertx-http".equals(artifactId));
```

Comment:
I converted `quarkus-messaging` entirely so we can drop this (I didn't do it then to avoid conflicts with your PR)!